docker/docker-compose.yml

```
@@ -265,9 +265,7 @@ services:
       # next.js needs build-time access to a number of API URL values, forward as ARGs
       args:
         # Web front-end URL
-        - WEB_URL
-        # Telescope 1.0 API URL
-        - API_URL
+        - WEB_URL=${WEB_URL}
```

I think `WEB_UR` is enough, we can just forward the args.

```suggestion
        - WEB_URL
```